scene/2d/animated_sprite.cpp

```diff
@@ -144,20 +144,24 @@ void SpriteFrames::clear(const StringName &p_anim) {
 void SpriteFrames::clear_all() {
 	animations.clear();
 	add_animation("default");
+	// emit_changed(); will be triggered by the previous line, no need to do it twice
```

Comment:
Makes it look less like dead code to remove.
```suggestion
	add_animation("default"); // Also emits changed.
```